### PR TITLE
Feature/bootstrap dropdown

### PR DESCRIPTION
**Pull request by pkhlopiachyi**

### CODE REVIEW

File: src/components/CustomInput/CustomInput.pcss [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  }
+
+  label {
+    background: var(--email-form-bg);
+    position: absolute;
+    top: 0px;
+    z-index: 5;
+    margin-top: calc(var(--gap) * (-2));
+  }
+}
```
Comment (josadcha) [inline src/components/CustomInput/CustomInput.pcss]: add empty line plz

File: src/components/Dropdown/index.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+interface DropdownComponentState {
+    selected: string;
+    selectedIndex: string;
+}
+
+/**
+ *  Cryptobase Dropdown that overrides default dropdown with list of options.
+ */
+class DropdownComponent extends React.Component<DropdownComponentProps & {}, DropdownComponentState> {
+
```
Comment (josadcha) [inline src/components/Dropdown/index.tsx]: remove line

File: src/components/Dropdown/index.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        );
+    }
+
+    private renderElem = (elem: DropdownElem, index: number) => {
+        return  (<Dropdown.Item
+                    onSelect={ (eventKey: any, e?: React.SyntheticEvent<unknown>) => this.handleSelect(elem, index)}
+                >
+                    {elem}
+                </Dropdown.Item>);
+    }
```
Comment (josadcha) [inline src/components/Dropdown/index.tsx]: fix spacing here, it'll look more readable like this:
```
return (
    <Dropdown.Item onSelect={ (eventKey: any, e?: React.SyntheticEvent<unknown>) => this.handleSelect(elem, index)}>
        {elem}
    </Dropdown.Item>
);
```

File: src/components/Dropdown/index.tsx [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+    private handleSelect = (elem: DropdownElem, index: number) => {
+        this.props.onSelect && this.props.onSelect(index);
+        this.setState({
+            selected: this.convertToString(elem),
+            selectedIndex: index.toString(),
+        });
+        window.console.log(this.state.selectedIndex);
```
Comment (josadcha) [inline src/components/Dropdown/index.tsx]: remove console log

File: src/components/SignIn/SignIn.pcss [hunk 5]
```diff
@@ -226,21 +144,5 @@
...
-      cursor: pointer;
-      opacity: 0.8;
-      text-align: center;
-
-      &:hover {
-        opacity: 1;
-      }
-    }
   }
 }
```
Comment (josadcha) [inline src/components/SignIn/SignIn.pcss]: empty line please

File: src/containers/Confirm/Identity/index.tsx [hunk 6]
```diff
@@ -172,19 +172,19 @@ class IdentityComponent extends React.Component<Props, IdentityState> {
...
+                                <input
+                                    className="pg-confirm__content-identity-col-row-content-number"
+                                    type="string"
+                                    placeholder={this.translate('page.body.kyc.identity.firstName')}
+                                    value={firstName}
+                                    onChange={this.handleChange('firstName')}
+                                    onFocus={this.handleFieldFocus('firstName')}
+                                    onBlur={this.handleFieldFocus('firstName')}
+                                    autoFocus={true}
+                                />
```
Comment (josadcha) [inline src/containers/Confirm/Identity/index.tsx]: why not to use CustomInput component here?

File: yarn.lock [hunk 7]
```diff
@@ -3627,7 +3627,15 @@ cryptocoins-icons@^2.8.0:
...
   version "0.16.1"
+<<<<<<< HEAD
+<<<<<<< HEAD
   resolved "https://registry.yarnpkg.com/cryptocurrency-icons/-/cryptocurrency-icons-0.16.1.tgz#33ec0b828c666f773b53866b4b803b2a06c28efc"
+=======
+  resolved "https://registry.npmjs.org/cryptocurrency-icons/-/cryptocurrency-icons-0.16.1.tgz#33ec0b828c666f773b53866b4b803b2a06c28efc"
+>>>>>>> Feature: replace use bootstrap input instead of openware input
+=======
+  resolved "https://registry.npmjs.org/cryptocurrency-icons/-/cryptocurrency-icons-0.16.1.tgz#33ec0b828c666f773b53866b4b803b2a06c28efc"
+>>>>>>> Feature: replace use bootstrap input instead of openware input
```
Comment (josadcha) [inline yarn.lock]: reset changes in yarn.lock

**Comment by pkucherenk0** [comment]
Selection_276

Selection_277

Selection_278

**Comment by pkucherenk0** [comment]
Selection_280

Selection_281

Selection_282

Selection_283

Selection_284

Selection_285

Selection_286

Selection_287

Selection_288

**Comment by pkucherenk0** [comment]
1. Labels for fields and field should not be on top of each other 
2. Input should be white
3. **Drop down on KYC scroll making whole table to scroll not drop down content**

**Pull request closed without merging**